README.md

```
@@ -319,6 +326,15 @@ ThreatFlux Anthropic SDK provides broad coverage of the Anthropic API:
 - ✅ Tool use (function calling)
 - ✅ System prompts
 - ✅ Conversation history
+- ✅ Server-side fallbacks, fallback credit, diagnostics, and user-profile attribution
+- ✅ Current code-execution, MCP, tool-search, and text-editor result blocks
+
+### Current beta APIs
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)